### ISSUE TITLE
Which images did you use for validation set?

### ISSUE THREAD

**Issue by luke2997** [vqdang/hover_net#70]
I noticed on other issues you mentioned you just picked images randomly for the validation set, but depending on the images I choose my classification score for miscellaneous goes from 0 to 0.13 to 0.3, so therefore it depends on which ones i pick, but I can't get the magic 0.42 in your report. Could you clarify which were selected for your publication?

**Comment by simongraham** [comment]
The performance of the miscellaneous class varies quite significantly from epoch to epoch. The nuclei here are under-represented and are often very small. Therefore it is very hard to obtain a high-level of performance. This category was effectively as a bit of a dumping ground for categories that did not fit well into the other defined categories. Therefore, I would not worry too much about not getting our sweet spot of 0.42. Try using all images in the training set and using the test as validation. Of course this may be over-fitting to the test set, but it can give you an idea on the model performance capability.
